modernizer-maven-plugin/src/main/resources/modernizer.xml

```diff
@@ -810,18 +810,6 @@ violation names use the same format that javap emits.
     <comment>Prefer java.nio.file.Files.readAllLines(java.nio.file.Path, java.nio.charset.Charset)</comment>
   </violation>
 
-  <violation>
```

Comment:
I think tool chain or some other mechinism to downcast compilation will be required to pull this test check off because the code physically doesn't exist in new jdks.  So if travis is configured to run higher jdks, it's just going to fail as a result of this.  After newer PR separations I raised are in, I'll look at how to accomplish this so the test can still be ran.  Easier way might be to upgrade to junit jupiter and then add annotation to only run this specific test scan with jdk8 but not others. but that might require reflective usage considering the fact that it will fail compilation otherwise on higher jdks.